syntax = "proto2";

package pymaid.pb;


message Controller {
    optional string service_name = 1;
    optional string method_name = 2;
    optional uint32 transmission_id = 3;
    optional bool from_stub = 4;
    optional bool is_canceled = 5;
    optional bool is_failed = 6;
    optional bytes message = 7;
    optional bool is_notification = 8;
}


message Void {
}


message ErrorMessage {
    required uint32 error_code = 1;
    required string error_message = 2;
}
